OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -329,6 +329,8 @@
     <Compile Include="Traits\Conditions\GrantExternalConditionToCrusher.cs" />
     <Compile Include="Traits\Conditions\GrantRandomCondition.cs" />
     <Compile Include="Traits\Contrail.cs" />
+    <Compile Include="Traits\Multipliers\ProductionCostMultiplier.cs" />
```

Comment:
Only the interface should stay in Common namespace, the convention deployed here still prefers that traits be shifted to AS namespace. See ResourcePurifier for an example.

Reply:
This should also involve an Authors.AS edit thenafter.